client/Log.cpp

```diff
@@ -86,6 +86,14 @@ std::string FormatPacket(const OP2Internal::Packet& packet)
 }
 
 
+std::string FormatHexadecimal(std::uintptr_t value)
+{
+	std::stringstream ss;
+	ss << "0x" << std::hex << value;
+	return ss.str();
+}

```

Comment:
Based on the parameter, this appears to be specifically for formatting address values. Maybe this and all related formatting functions could be called `FormatAddress`.

----

Address values are often given in a fixed width format. Perhaps we should set field padding such that the output is always the same length. Ideally this can be based on the architecture, though as this project can only ever be compiled for 32-bit, I would be fine with a hardcoded output length of 8 characters (plus optionally the `0x` prefix).

Reply:
Good idea. Since the load addresses already displayed with 8 characters, I hadn't thought of this.